File: app/src/main/java/tech/salroid/filmy/customs/CustomToast.java
package tech.salroid.filmy.customs;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;
import android.widget.Toast;
import tech.salroid.filmy.R;

/*
 * Filmy Application for Android
 * Copyright (c) 2016 Ramankit Singh (http://github.com/webianks).
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

public class CustomToast {

    public static void show(Context context, String message,boolean showImage){


        if(context!=null){

            LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);

            View layout = inflater.inflate(R.layout.custom_toast, null);

            TextView text = (TextView) layout.findViewById(R.id.text);
            text.setText(message);

            ImageView imageView = (ImageView) layout.findViewById(R.id.image);

            if (!showImage)
            imageView.setVisibility(View.GONE);

            layout.requestLayout();

            Toast toast = new Toast(context.getApplicationContext());
            toast.setDuration(Toast.LENGTH_LONG);
            toast.setView(layout);
            toast.show();
        }



    }
}

File: app/src/main/java/tech/salroid/filmy/views/CustomToast.kt
package tech.salroid.filmy.views

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import android.widget.Toast
import tech.salroid.filmy.R

object CustomToast {

    @JvmStatic
    fun show(context: Context?, message: String?, showImage: Boolean) {

        val inflater = context?.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
        val layout = inflater.inflate(R.layout.custom_toast, null)
        val text = layout.findViewById<View>(R.id.text) as TextView
        text.text = message

        val imageView = layout.findViewById<View>(R.id.image) as ImageView
        if (!showImage) imageView.visibility = View.GONE
        layout.requestLayout()

        val toast = Toast(context.applicationContext)
        toast.duration = Toast.LENGTH_SHORT
        toast.view = layout
        toast.show()
    }
}